src/main/resources/assets/clientcommands/lang/en_us.json

```diff
@@ -151,6 +151,9 @@
   "enchCrack.insn.itemThrows.noDummy": "No dummy enchantment",
   "enchCrack.insn.enchantments": "Enchantments on item:",
 
+  "paginator.display.page": "Page number: %d of %d",
+  "paginator.incorrect.page": "Specified page is out of bounds",
```

Comment:
These should be grouped and named like `chelp` translation keys.

Reply:
Paginator was written not only for `/chelp` but also for other possible use cases.